__init__.py

```diff
@@ -183,12 +195,28 @@ def weather_at_location(self, name):
                 return self.weather_at_location(name)
             raise
 
+    def check_for_city_state(self, name):
+        # someone might ask for denver colorado (which would need to be written denver,colorado)
+        reg = self.owm.city_id_registry()
+        if len(reg.ids_for(name)) == 0:
+            # Try to rewrite name -> replace last blank with comma # TODO: careful this might be language specific -> check
+            p = name.rfind(" ")
+            if p>0:
+                name = name[:p] + "," + name[p+1:]
+        return name
+
     def weather_at_place(self, name, lat, lon):
         if lat and lon:
+            if self.owm is not None:
+                return self.owm.weather_at_coords(lat, lon)
             q = {"lat": lat, "lon": lon}
         else:
             if name in self.location_translations:
                 name = self.location_translations[name]
+            LOG.info("searching weather for {}".format(name)) # TODO: remove debug
```

Comment:
Best to switch this to debug level or remove before we merge. We aim to keep the info log stream as clean as possible.

Reply:
oops, wanted to remove that before PR, but maybe keeping it in as debug makes sense? Could even add some more debug logs?